packages/testlab/src/test-sandbox.ts

```diff
@@ -119,4 +120,17 @@ export class TestSandbox {
     await ensureDir(destDir);
     return writeJson(dest, data, {spaces: 2});
   }
+
+  /**
+   * Creates a new file and writes the given data as text.
+   *
+   * @param dest - Destination filename, optionally including a relative path.
+   * @param data - The data to write.
```

Comment:
```suggestion
   * @param data - The text to write.
```

Should we rename the `data` argument to `content` or `text`?